randovania/gui/ui_files/preset_msr_patches.ui

```diff
@@ -180,6 +180,26 @@
              </property>
             </widget>
            </item>
+           <item>
+            <widget class="QCheckBox" name="reverse_area8_check">
+             <property name="text">
+              <string>Reverse Area 8</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QLabel" name="reverse_area8_label">
+             <property name="mouseTracking">
+              <bool>true</bool>
+             </property>
+             <property name="text">
+              <string>Removes the wall between the Queen and Baby, allowing for earlier access into Area 8 by enerting the area backwards</string>
```

Comment:
`enerting` is not in my vocabulary and also don't show up in my dictionary. `entering` ?